openedx/adg/lms/applications/test/test_serializers.py

```diff
@@ -0,0 +1,122 @@
+"""
+All tests for applications APIs serializers
+"""
+import pytest
+
+from student.tests.factories import UserFactory
+
+from ..models import Education
+from ..serializers import EducationSerializer, WorkExperienceSerializer
+from .factories import UserApplicationFactory
+
+
+@pytest.fixture
+def user_application(request):
+    """
+    Create user with profile, this fixture will be passed as a parameter to all pytests
+    """
+    user = UserFactory()
+    return UserApplicationFactory(user=user)
+
+
+@pytest.fixture
+def education_data(request, user_application):  # pylint: disable=redefined-outer-name
+    """
+    Data for education serializer
+    """
+    return {
+        'date_started_month': 2,
+        'date_started_year': 2017,
+        'date_completed_month': 3,
+        'date_completed_year': 2018,
+        'name_of_school': 'PUCIT',
+        'degree': Education.BACHELOR_DEGREE,
+        'is_in_progress': True,
+        'user_application': user_application.id
+    }
+
+
+@pytest.fixture
+def work_experience_data(request, user_application):  # pylint: disable=redefined-outer-name
+    """
+    Data for education serializer
+    """
+    return {
+        'date_started_month': 2,
+        'date_started_year': 2018,
+        'date_completed_month': 8,
+        'date_completed_year': 2020,
+        'name_of_organization': 'Arbisoft',
+        'job_position_title': 'SSE',
+        'is_current_position': True,
+        'job_responsibilities': 'Testing',
+        'user_application': user_application.id
+    }
+
+
+@pytest.mark.django_db
+def test_education_serializer_with_valid_and_invalid_data(education_data):  # pylint: disable=redefined-outer-name
```

Comment:
Can we please split this test into multiple tests? Ideally we should have one assertion per test.
Also we can name a fixure differently to avoid `redefined-outer-name` issue.